code/game/jobs/job/security.dm

```diff
@@ -361,7 +361,7 @@
 	supervisors = "the Blackshield Commander"
 	difficulty = "Hard."
 	noob_name = "Blackshield Cadet"
-	alt_titles = list("Blackshield Cadet","Blackshield Militiamen")
+	alt_titles = list("Blackshield Cadet", "Blackshield Heavy Trooper", "Blackshield Designated Marksman", "Blackshield Recon Specialist")
```

Comment:
```suggestion
	alt_titles = list("Blackshield Cadet")
```
No additional specializations for it, please. No further 'This is my job!' than already exists.